### PR TITLE
JENKINS-46812 Test results always show duration as "A few seconds"

### PR DESCRIPTION
**Pull request by NicuPascu**
The humanize method that was used before would cause any value unde 60 sec to appear as "a few seconds", this changes this to the format we use in other parts of blueocean  

# Description

See [JENKINS-46812](https://issues.jenkins-ci.org/browse/JENKINS-46812).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by i386** [comment]
LGTM

File: blueocean-dashboard/src/main/js/components/testing/TestCaseResultRow.jsx [hunk 1]
```diff
@@ -24,7 +25,7 @@ export default class TestCaseResultRow extends Component {
     render() {
         const { testCase: t, translation, locale = 'en' } = this.props;
         moment.locale(locale);
-        const duration = moment.duration(Number(t.duration), 'milliseconds').humanize();
+        const duration = moment.duration(Number(t.duration), 'milliseconds').format('d[d] h[h] m[m] s[s]');
```
Comment (michaelneale) [inline blueocean-dashboard/src/main/js/components/testing/TestCaseResultRow.jsx]: is this localised - and/or the same as everywhere else?

**Comment by michaelneale** [comment]
right, the format from what @imeredith said would be good

File: jenkins-design-language/src/js/components/TimeDuration.jsx [hunk 2]
```diff
@@ -101,16 +116,19 @@ export class TimeDuration extends Component {
...
             } = this.props;
+
+            const liveFormat = t('common.date.duration.format', { defaultValue: 'm[ minutes] s[ seconds]' });
+            const hintFormat = t('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' });
+
             moment.locale(locale);
             // in case we are in live update we are interested in seconds
             const duration = this.props.liveUpdate ?
-                moment.duration(millis).format(liveFormat) : moment.duration(millis).format(displayFormat);
+                moment.duration(millis).format(liveFormat) : TimeDuration.format(millis, t, locale);
```
Comment (imeredith) [inline jenkins-design-language/src/js/components/TimeDuration.jsx]: Pretty sure that moment.duration is no longer needed here as liveformat and display format at the same. At the first lesat your format function should be the one to deal with the live format so it takes in <1s into account.

Comment (i386) [reply jenkins-design-language/src/js/components/TimeDuration.jsx]: @imeredith are you talking about the different formats here?
screen shot 2017-10-17 at 7 43 49 am

**Pull request merged**